src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -57,6 +56,7 @@ impl LazyBatchColumnVec {
     ///
     /// It is caller's duty to ensure that these columns are not decoded and the number of datums
     /// matches the number of columns. Otherwise there will be panics.
+    // TODO: Remove this function, since it is not used at all.
```

Comment:
Maybe use `  #[cfg(test)]` instead?

Reply:
Not just `#[cfg(test)]`, it can be removed totally, but we will then need to update a lot of tests. We can leave contributors to do it later 😄😏